src/main/java/org/cbioportal/persistence/StudyViewRepository.java

```diff
@@ -27,7 +27,7 @@ public interface StudyViewRepository {
     
     List<ClinicalData> getPatientClinicalData(StudyViewFilterContext studyViewFilterContext, List<String> attributeIds);
     
-    List<AlterationCountByGene> getMutatedGenes(StudyViewFilterContext studyViewFilterContext);
+    List<AlterationCountByGene> getMutatedGenes(StudyViewFilterContext studyViewFilterContext, String specificHugoGeneSymbol);
```

Comment:
@haynescd @fuzhaoyuan is there a way to indicate that this is optional rather than forcing people to pass null?  I think that's called overloading at method, right?

Reply:
Also, I think maybe just "hugoGeneSymbol" is enough.  specificHugGeneSymbol seems redundant.   There should be a comment explaining that we pass in a genesymbol if we only want counts for that gene.   i also wonder whether we should just make it an array since it seems inevitable that we'll want to target multiple genes.

Reply:
I would honestly create a new method called getMutatedGene that would reuse the sql in the mapper file for getMutatedGenes...

Kinda doesn't make sense to have getMutatedGenes that returns one gene

Reply:
Yeah I'll look into it

Reply:
maybe two methods (they could be overloaded )
getMutatedGenes
getMutatedGenes(StudyViewFilter, String... geneToQuery)